packages/devtools_app/lib/src/primitives/utils.dart

```diff
@@ -0,0 +1,1422 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
```

Comment:
nit: 2022

Can you also add a todo somewhere at the top of this file (but below the copyright) to move this code back to shared/utils.dart once the null safety migration is complete? Maintaining two utils files doesn't seem necessary moving forward.

Reply:
Discussed offline and agreed to keep two utils.dart: higher level and lower level.